### PR TITLE
Ignore .swp files

### PR DESCRIPTION
**Pull request by rnestler**
Check the following:

 - [na] Breaking changes marked in commit message
 - [na] Changelog updated
 - [na] Code style cleaned (ran `make style-fix`)
 - [na] Tested on actual hardware

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+*.swp
```
Comment (abrauchli) [inline .gitignore:1]: That's editor-specific and IMHO belongs in your local `~/.gitignore` (same as `tags`, `*.swo` etc.)

**Review by abrauchli — CHANGES_REQUESTED** [review]
I'm tilting towards NACK.. but open for discussions

File: .gitignore (hunk 1 above)
Comment (rnestler) [reply .gitignore:1]: Huh. Funny enough I *have* `*.swp` in my `~/.gitignore` but git seems to ignore it.

Comment (rnestler) [reply .gitignore:1]: Reading https://git-scm.com/docs/gitignore it seems that the global ignore file resides in `$XDG_CONFIG_HOME/git/ignore`

Comment (rnestler) [reply .gitignore:1]: But fair enough.

**Comment by rnestler** [comment]
See review comment.

**Pull request closed without merging**